packages/riipen-ui/src/components/RadioButtonGroup.test.jsx

```diff
@@ -0,0 +1,178 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import RadioButtonGroup from "./RadioButtonGroup";
+import RadioButton from "./RadioButton";
+
+describe("<RadioButtonGroup>", () => {
+  it("renders without errors", () => {
+    let error;
+    const child = <RadioButton />;
+
+    try {
+      mount(<RadioButtonGroup>{child}</RadioButtonGroup>);
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const child = <RadioButton />;
+
+    const wrapper = mount(<RadioButtonGroup>{child}</RadioButtonGroup>);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("children prop", () => {
+    it("renders non null children", () => {
+      const child = <RadioButton id="one" />;
+
+      const wrapper = mount(
+        <RadioButtonGroup>
+          {null}
+          {child}
+        </RadioButtonGroup>
+      );
+
+      const childrenNodes = wrapper
+        .find("fieldset")
+        .find("span")
+        .children();
+
+      expect(childrenNodes.at(0).props().id).toEqual("one");
+    });
+  });
+
+  describe("error prop", () => {
+    it("displays error", () => {
```

Comment:
Wondering if this needs to be a more descriptive test...something like "displays given error message"?